pkg/operator/removelatencysensitive/removelatencysensitive_controller.go

```diff
@@ -0,0 +1,76 @@
+package removelatencysensitive
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	configv1 "github.com/openshift/api/config/v1"
+	applyconfigurationsconfigv1 "github.com/openshift/client-go/config/applyconfigurations/config/v1"
+	configv1client "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1"
+	v1 "github.com/openshift/client-go/config/informers/externalversions/config/v1"
+	configlistersv1 "github.com/openshift/client-go/config/listers/config/v1"
+	"github.com/openshift/library-go/pkg/controller/factory"
+	"github.com/openshift/library-go/pkg/operator/events"
+	operatorv1helpers "github.com/openshift/library-go/pkg/operator/v1helpers"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// LatencySensitiveRemovalController clears the LatencySensitive featuregate value
+type LatencySensitiveRemovalController struct {
+	featureGatesClient configv1client.FeatureGatesGetter
+	featureGatesLister configlistersv1.FeatureGateLister
+
+	eventRecorder events.Recorder
+}
+
+func NewLatencySensitiveRemovalController(operatorClient operatorv1helpers.OperatorClient,
+	featureGatesClient configv1client.FeatureGatesGetter, featureGatesInformer v1.FeatureGateInformer,
+	eventRecorder events.Recorder) factory.Controller {
+	c := &LatencySensitiveRemovalController{
+		featureGatesClient: featureGatesClient,
+		featureGatesLister: featureGatesInformer.Lister(),
+		eventRecorder:      eventRecorder,
+	}
+
+	return factory.New().
```

Comment:
intentionally not wired to an informer to avoid rapid fighting. This will be once a minute.